site/content/en/latest/user/basic-auth.md

```diff
@@ -21,6 +21,55 @@ Envoy Gateway uses [.htpasswd][.htpasswd] format to store the username-password
 The file must be stored in a kubernetes secret and referenced in the [SecurityPolicy][SecurityPolicy] configuration. 
 The secret is an Opaque secret, and the username-password pairs must be stored in the key ".htpasswd".
 
+Note: We recommend users to enable HTTPS when using Basic Authentication.
+
```

Comment:
nice ! only one nit: can we move this diff before L28, so the `.htpasswd` sentence is below this ?

Reply:
👌